scripts/billing/sql/daily_compute_cores_average.sql

```diff
@@ -0,0 +1,17 @@
+/*
+Query returns number of cores executed on average on a daily basis
+*/
+
+SELECT
+  FORMAT_DATETIME("%Y%m%d", DATETIME(usage_end_time)) AS date_ymdh, 
+  AVG(CAST(system_labels.value as int64)) AS avg_core_usage
+FROM
+  `data-analytics-pocs.public.billing_dashboard_export`,
+  UNNEST(system_labels) AS system_labels
+WHERE 
+  service.id = "6F81-5844-456A"
```

Comment:
Can you add 2 spaces after the quotes and a single-line comment describing what this identifier is?

Example:
```sql
WHERE 
  service.id = '6F81-5844-456A'  -- Compute Engine
```